framework/src/problems/FEProblemBase.C

```diff
@@ -4214,9 +4224,18 @@ void
 FEProblemBase::execMultiAppTransfers(ExecFlagType type, Transfer::DIRECTION direction)
 {
   bool to_multiapp = direction == MultiAppTransfer::TO_MULTIAPP;
-  std::string string_direction = to_multiapp ? " To " : " From ";
-  const MooseObjectWarehouse<Transfer> & wh =
-      to_multiapp ? _to_multi_app_transfers[type] : _from_multi_app_transfers[type];
+  bool from_multiapp = direction == MultiAppTransfer::FROM_MULTIAPP;
+  std::string string_direction;
+  if (to_multiapp)
+    string_direction = " To ";
+  else if (from_multiapp)
+    string_direction = " From ";
+  else
+    string_direction = " between ";
```

Comment:
Should we use enum for the direction instead of string?

We could move the following enum to the MooseTypes.h

```
  enum DIRECTION
  {
    TO_MULTIAPP,
    FROM_MULTIAPP
    FROM_MULTIAPP,
    BETWEEN_MULTIAPP
  };
```

Reply:
this isn't the location for it though, this one is just for a log if I recall